packages/sil-governance/src/routes/create-proposal/components/TitleField.tsx

```diff
@@ -11,14 +13,29 @@ import React from "react";
 
 export const TITLE_FIELD = "Title";
 const TITLE_PLACEHOLDER = "Enter Title";
-const TITLE_MAX_LENGTH = 30;
+const TITLE_MIN_LENGTH = 10;
+const TITLE_MAX_LENGTH = 50;
 
 interface Props {
   readonly form: FormApi;
 }
 
 const TitleField = ({ form }: Props): JSX.Element => {
-  const validator = composeValidators(required, notLongerThan(TITLE_MAX_LENGTH));
+  const charsetValidator = (value: FieldInputValue): string | undefined => {
+    if (typeof value !== "string") throw new Error("Input must be a string");
+
+    const isCharsetValid = !!value.match(/^[a-zA-Z0-9 _.-]*$/);
+
+    if (isCharsetValid) return undefined;
+    return `Symbols supported: ".", "-", "_"`;
+  };
+
+  const validator = composeValidators(
+    required,
+    longerThan(TITLE_MIN_LENGTH),
```

Comment:
Side note: this is a broken naming for a min length validator since "longer that 5" will require >= 5, not > 0. Anyways, this is out of scope here